microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -77,6 +77,26 @@ When you visit a webpage and submit a form, Microsoft Edge sends information abo
 
 In **Guest** mode, autofill is not available and new autofill entries aren't added.  For **InPrivate** mode, Microsoft Edge offers autofill entries, but new autofill entries aren't added.
 
+<!-- ====================================================================== -->
+## Bing Chat
+
+Bing chat is an AI-enhanced web search experience. It supports users by summarizing web search results and providing a chat experience. Users can also generate creative content, such as poems, jokes, and letters. The new AI-enhanced Bing runs on a variety of advanced technologies from Microsoft and OpenAI, including GPT-4, a cutting-edge large language model (LLM) from OpenAI. 
```

Comment:
```suggestion
Bing Chat is an AI-enhanced web search experience. It supports users by summarizing web search results and providing a chat experience. Users can also generate creative content, such as poems, jokes, and letters. The new AI-enhanced Bing runs on a variety of advanced technologies from Microsoft and OpenAI, including GPT-4, a cutting-edge large language model (LLM) from OpenAI.
```
* Uppercase "Chat" - imo we're talking about a feature, that's presented mainly as just **Chat** in the UI.